integrations/instructor_embedders/pydoc/config.yml

```diff
@@ -0,0 +1,30 @@
+loaders:
+  - type: haystack_pydoc_tools.loaders.CustomPythonLoader
+    search_path: [../src]
+    modules:
+      [
+        "haystack_integrations.components.embedders.instructor_embedders",
+        "haystack_integrations.components.embedders.instructor_embedders.embedding_backend",
+      ]
+    ignore_when_discovered: ["__init__"]
+processors:
+  - type: filter
+    expression:
+    documented_only: true
+    do_not_filter_modules: false
+    skip_empty_modules: true
+  - type: smart
+  - type: crossref
+renderer:
+  type: haystack_pydoc_tools.renderers.ReadmePreviewRenderer
+  excerpt: Instructor embedding integration for Haystack
```

Comment:
embedders? Here and below, perhaps we see what other integrations wrote

Reply:
Embedders sounds good to me. 👍 

Other integrations are more to the point with naming since they use a product name.